hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyService.java

```diff
@@ -867,19 +878,20 @@ public static class RequestInterceptorChainWrapper {
     /**
      * Initializes the wrapper with the specified parameters.
      * 
-     * @param rootInterceptor the root request intercepter
+     * @param rootInterceptor the root request interceptor
      * @param applicationAttemptId attempt id
      */
+    @SuppressWarnings("checkstyle:HiddenField")
```

Comment:
Can we rename the parameter to "rootIntercept" and "appAttemptId"?

Reply:
I read the code and I think there is no problem with the variable naming and methods, the problem is on the HiddenField check in checkstyle, I added a xml configuration that doesn't check for variables that contain init characters.

```
<module name="HiddenField">
    -- add configuration
    <property name="ignoreFormat" value="init"/>
    <property name="ignoreConstructorParameter" value="true"/>
    <property name="ignoreSetter" value="true"/>
</module>
```

Is it possible for me to do this?

Reply:
I think the checkstyle warning is legit.
The code is easier to read if the variable name is different.
Adding the "this" makes it correct but it hides the high-level issue.
I would rename the arguments to make clear that they are different than the fields.

Reply:
Thank you for your suggestion, your suggestion makes sense, I will modify the code.